package.json

```diff
@@ -26,6 +26,8 @@
     "@types/mongoose": "^5.11.97",
     "@types/nconf": "^0.10.1",
     "@types/node": "^16.7.13",
+    "@types/swagger-ui-express": "^4.1.3",
+    "@types/yamljs": "^0.2.31",
```

Comment:
I think this is not needed anymore ?

Reply:
removed 7d8e077a4e93dd9df4885d6bb22eb0c9a4ba657c